dev.sh

```diff
@@ -4,7 +4,7 @@ docker-compose down
 
 cat env.example > .env
 
-cd ./cli
+cd ./cli || exit
```

Comment:
Do we want it to exit without error?

Reply:
It doesn't exit without an error earlier it wouldn't have exited at all rather further process would have been tried in the wrong directory. Above shall exit while returning the error from cd